dataactcore/models/lookups.py

```diff
@@ -79,7 +79,8 @@
     LookupFileType(5, 'award_procurement', '', 'D1', 5, True, AwardProcurement),
     LookupFileType(6, 'executive_compensation', '', 'E', None, False, None),
     LookupFileType(7, 'sub_award', '', 'F', None, False, None),
-    LookupFileType(8, 'fabs', '', 'FABS', None, False, FABS)
+    LookupFileType(8, 'fabs', '', 'FABS', None, False, FABS),
+    LookupFileType(9, 'boc_comparison', '', 'BOC', None, False, None)
```

Comment:
Spent some time looking around for any other instances where `FILE_TYPE_DICT`/`FILE_TYPE_DICT_ID` as a whole list is used. There are _some_ instances where it could show up but nothing breaking or advertised, so i think it's safe.